config.go

```diff
@@ -41,6 +42,7 @@ type config struct {
 	ErrorTemplate     *string
 	SuccessTemplate   *string
 	FailureTemplate   *string
+	Timeout           int
```

Comment:
Believe this needs to be an int pointer or else configs without this will fail.  Test that thought as I'm not sure.